src/resources/library/forms_db_controller.php

```diff
@@ -181,20 +183,23 @@ public function renderFormsList()
         echo "</tr>";
         echo "</thead>";
         echo "<tbody>";
+
         foreach ($form as $row) {
           echo "<tr>";
           if (empty($row["FormXml"])){
             echo "<td></td>";
           } else {
-            echo "<td>" . "<a href='#' onclick='onPreviewClick(\"" . $row["FormName"] . "\",\"" . $row["Description"] . "\",\"" . $row["FormXml"] . "\")'>Preview</a></td>";            
+            echo "<td>" . "<a href='#' onclick='onPreviewClick(\"" . $row["FormName"] . "\",\"" . $row["Description"] . "\",\"" . $row["FormXml"] . "\")' class='btn btn-primary'>Preview</a></td>";            
           }
           echo "<td>" . $row["FormName"] . "</td>";
           echo "<td>" . $row["Description"] . "</td>";
           $formid= $row["id"];
           $buildformid = "buildform" . $formid;
-          echo "<td><form id='$buildformid' method='post' action='formsbuild.php'><input name='id' type='hidden' value='$formid'><a href='' onclick=\"document.getElementById('$buildformid').submit();return false;\">Edit</a></form></td>";
+          echo "<td><form id='$buildformid' method='post' action='formsbuild.php'><input name='id' type='hidden' value='$formid'><a href='' onclick=\"document.getElementById('$buildformid').submit();return false;\" class='btn btn-success'>Edit</a></form></td>";
           $delformid = "delform" . $formid;            
-          echo "<td><form id='$delformid' method='post'><input name='id' type='hidden' value='$formid'><input name='deleteform' type='hidden'><a href='' title='Delete Form' onclick=\"document.getElementById('$delformid').submit();return false;\">X</a></form></td>";
+
+		  echo "<td><form id='$delformid' method='post'><input name='id' type='hidden' value='$formid'>                
+<input name='deleteform' type='hidden'><a href='' title='Delete Form' onclick=\"document.getElementById('$delformid').submit();return false;\" class='btn btn-danger'>X".$pg_encrypt_key."</a></form></td>";
```

Comment:
No it should not be... Im bad about that lol

Sent from my iPhone

> On Jul 25, 2016, at 5:21 PM, Raymond Brady <notifications@github.com> wrote:
> 
> In src/resources/library/forms_db_controller.php:
> 
> >            $delformid = "delform" . $formid;            
> > -          echo "<td><form id='$delformid' method='post'><input name='id' type='hidden' value='$formid'><input name='deleteform' type='hidden'><a href='' title='Delete Form' onclick=\"document.getElementById('$delformid').submit();return false;\">X</a></form></td>";
> > +
> > +		  echo "<td><form id='$delformid' method='post'><input name='id' type='hidden' value='$formid'>                
> > +<input name='deleteform' type='hidden'><a href='' title='Delete Form' onclick=\"document.getElementById('$delformid').submit();return false;\" class='btn btn-danger'>X".$pg_encrypt_key."</a></form></td>";
> Does $pg_encrypt_key need to be echoed here?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>